CameraManip/src/CameraManipPlugin.cpp

```diff
@@ -88,6 +92,7 @@ void CameraManipPluginC::useSelectedCamera() {
             Ra::Engine::Camera* camera = static_cast<Ra::Engine::Camera*>( ent.m_component );
             m_viewer->getCameraManipulator()->getCamera()->show(true );
             m_viewer->getCameraManipulator()->setCamera(camera );
+            askForUpdate();
```

Comment:
As `askForUpdate()` is a signal, use `emit askForUpdate()` instead of just calling directly the method.

Reply:
well this one will call `Ra::Plugins::Context::askForUpdate()` since a signal is not callable.
Not sure which one is the best practice.

Reply:
```suggestion
            emit askForUpdate();
```

Reply:
> well this one will call `Ra::Plugins::Context::askForUpdate()` since a signal is not callable.
> Not sure which one is the best practice.

You can call a signal. This translate to calling the moc-generating code but is less "readable" than emiting it.

Reply:
Right (I haven't updated since the old SIGNAL, SLOT mechanism ...)